File: Python/07-most-letters.py
def nearby_az(string):

    for i in range(0, len(string)):
        if string[i] == 'a':
            for j in range(i, i+4):
                try:
                    char = string[j]
                    if char == 'z':
                        return True
                except IndexError:
                    continue

    return False

print(nearby_az("zabsz"))

File: Javascript/07-most-letters.js
function nearbyAz(string) {

	for (var i = 0; i < string.length-1; i++) {
		if (string[i] == 'a' &&
		   (string[i+1] == 'z' ||
			string[i+2] == 'z' ||
			string[i+3] == 'z')) {
			return true;
		}
	}

	return false;
}

console.log(nearbyAz("zebceaz"));
